airplan/assets/page.css

```
@@ -251,6 +251,67 @@ body.toc-dialog-open {
+  align-items: center;
+  gap: 0.4rem;
+  border-radius: 6px;
+  cursor: pointer;
+}
+
+.revision-heading.is-picker::after {
+  content: "";
+  width: 6px;
+  height: 6px;
+  flex: none;
+  border-right: 1.5px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)